extensions/ql-vscode/src/view/common/SuggestBox/SuggestBox.tsx

```diff
@@ -36,7 +35,6 @@ const Input = styled(VSCodeTextField)<{ $error: boolean }>`
 `;
 
 const Container = styled.div`
-  width: 430px;
   display: flex;
```

Comment:
This was a styling fix to stop the model editor looking like this 😅

![image](https://github.com/github/vscode-codeql/assets/42641846/435ec5f2-94ae-490d-a0c0-1a07efbb0404)